docgenerator/SDKDocGeneratorLib/Templates/sdk-api-home.html

```diff
@@ -172,91 +172,42 @@ <h2 class="title" style="clear: both" id="d0e153">
 
                             <dt>
                                 <span class="term">
-                                    <a class="link" href="../v3-migration.html" target="_blank" title="V3 Migration Guide">
-                                        V3 Migration Guide
+                                    <a class="link" href="../v4-migration.html" target="_blank" title="V4 Migration Guide">
+                                        V4 Migration Guide
                                     </a>
                                 </span>
                             </dt>
 
                             <dd>
-                                <p>Guide to help developers move their applications from version 2 to version 3 of the AWS SDK for .NET.</p>
+                                <p>Guide to help developers move their applications from version 3 to version 4 of the AWS SDK for .NET.</p>
                             </dd>
 
                             <dt>
                                 <span class="term">
-                                    <a class="link" href="https://docs.aws.amazon.com/sdk-for-net/latest/developer-guide/net-dg-v35.html" target="_blank" rel="noopener noreferrer" title="V3.5 Migration Guide">
-                                        V3.5 Migration Guide
+                                    <a class="link" href="https://docs.aws.amazon.com/sdk-for-net/latest/developer-guide/net-dg-v4.html" target="_blank" rel="noopener noreferrer" title="V4.0 Migration Guide">
+                                        V4.0 Migration Guide
                                     </a>
                                 </span>
                             </dt>
 
                             <dd>
-                                <p>Guide to help developers move their applications from version 3.3 to version 3.5 of the AWS SDK for .NET.</p>
+                                <p>Guide to help developers move their applications using AWS SDK for .NET Version 3 targeting .NET Framework to a modern version of .NET.</p>
                             </dd>
 
                             <dt>
                                 <span class="term">
-                                    <a class="link" href="https://docs.aws.amazon.com/sdk-for-net/latest/developer-guide/migration-from-net-standard-1-3.html" target="_blank" rel="noopener noreferrer" title="Migrating from .NET Standard 1.3">
-                                        Migrating from .NET Standard 1.3
+                                    <a class="link" href="https://docs.aws.amazon.com/sdk-for-net/latest/developer-guide/migration-from-net-standard-1-3.html" target="_blank" rel="noopener noreferrer" title="Migrating from .NET Framework 3.5">
```

Comment:
The title has been changed to .NET Framework 3.5 but the link is still to a page for migrating from .NET Standard 1.3. Is this a placeholder change? Should we just remove this block till we have migration guide?

Reply:
It was a placeholder as we should have this guide but I don't have a link. There are actually a few placeholders on this page as we don't have a V4.0 Migration Guide yet either. 

I can remove these though if we want to revisit them later.

Reply:
I would suggest removing instead of leaving links that point to wrong content for the link title.